jupyter_server/serverapp.py

```diff
@@ -511,7 +511,12 @@ def shutdown_server(server_info, timeout=5, log=None):
     if log:
         log.debug("POST request to %sapi/shutdown", url)
 
-    fetch(url, method="POST", body=b"", headers={"Authorization": "token " + server_info["token"]})
+    fetch(
+        urljoin(url, "api/shutdown"),
```

Comment:
Excellent catch @dleen  - thank you!

My only comment would be that the url is composed prior to the debug so that what we're logging is the truth.

Reply:
Oh great point - the debug log saying the api/shutdown endpoint when it wasn't using that at all was very confusing